### PR TITLE
Add new blog post for OTel Q&A - May

### PR DESCRIPTION
**Pull request by avillela**
Preview: https://deploy-preview-2812--opentelemetry.netlify.app/blog/2023/end-user-q-and-a-03/

### CODE REVIEW

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+Traces are being collected through a combination of manual and auto
+instrumentation.
+
+Some applications are being manually instrumented through OpenTelemetry, and
+others are still being instrumented using OpenTracing,
+[which is backwards-compatible with OpenTelemetry](https://opentelemetry.io/docs/migration/opentracing/).
+
+The OpenTelemetry Operator is being implemented to auto-instrument Java and .NET
+code. The OTel Operator
+([among other things](https://opentelemetry.io/docs/k8s-operator/#introduction)`)
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:161]: Drop lost tick character (at least to fix the GH syntax highlighting :))

```suggestion
([among other things](https://opentelemetry.io/docs/k8s-operator/#introduction))
```

**Comment by avillela** [comment]
> Replace the following external links by (local) paths:
> 
> * `https://opentelemetry.io/docs/...` by `/docs/...`
> * `https://opentelemetry.io/community/...` by `/community/...`

Done @chalin. I thought there used to be a linter that would catch this? I don't recall it erring out this time.

**Comment by chalin** [comment]
> Done @chalin. 

👍 

> I thought there used to be a linter that would catch this? I don't recall it erring out this time.

We decided to turn that check into a warning. To see those warnings you (currently) have to look at the build log, e.g.: https://github.com/open-telemetry/opentelemetry.io/actions/runs/5149237070/jobs/9271985379:

```
Start building sites … 
hugo v0.111.3-5d4eb5154e1fed125ca8e9b5a0315c4180dab192+extended linux/amd64 BuildDate=2023-03-12T11:40:50Z VendorInfo=gohugoio
WARN 2023/06/01 21:07:44 blog/2023/end-user-q-and-a-03.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/community/end-user/interviews-feedback/
WARN 2023/06/01 21:07:44 blog/2023/end-user-q-and-a-03.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/docs/concepts/signals/traces/
WARN 2023/06/01 21:07:44 blog/2023/end-user-q-and-a-03.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/docs/concepts/signals/metrics/
WARN 2023/06/01 21:07:44 blog/2023/end-user-q-and-a-03.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/docs/concepts/signals/
WARN 2023/06/01 21:07:44 blog/2023/end-user-q-and-a-03.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/docs/collector/
WARN 2023/06/01 21:07:[44](https://github.com/open-telemetry/opentelemetry.io/actions/runs/5149237070/jobs/9271985379#step:4:45) blog/2023/end-user-q-and-a-03.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/docs/migration/opentracing/
WARN 2023/06/01 21:07:44 blog/2023/end-user-q-and-a-03.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/docs/k8s-operator/#introduction
...
```

I'll see if I can surface those to make it easier --> [issue TBD].

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+challenges that they face, and to share it with the community, so that together,
+we can help make OpenTelemetry awesome!
+
+This month, I spoke with
+[Iris Dyrmishi](https://www.linkedin.com/in/iris-dyrmishi-b15a9a164/), Platform
+Engineer at [Farfetch](https://www.farfetch.com).
+
+## Overview
+
+Iris is a huge fan of Observability and OpenTelemetry, and her love of these two
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:27]:
```suggestion
Iris is a huge fan of [observability](/docs/concepts/observability-primer/) and OpenTelemetry, and her love of these two
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+[OpenTelemetry Collector](/docs/collector/) to gather and send metrics and
+traces to various Observability back-ends. Since she and her team started using
+OpenTelemetery, they started instrumenting more traces. In fact, with their
+current setup, Iris has happily reported that they went from processing 1,000
+spans per second, to processing 40,000 spans per second!
+
+### How are you collecting your traces right now?
+
+Traces are being collected through a combination of manual and auto
+instrumentation.
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:151]:
```suggestion
Traces are being collected through a combination of [manual and auto
instrumentation](/docs/concepts/instrumenting/).
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+
+Some applications are being manually instrumented through OpenTelemetry, and
+others are still being instrumented using OpenTracing,
+[which is backwards-compatible with OpenTelemetry](/docs/migration/opentracing/).
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:155]:
```suggestion
others are still instrumented using the [legacy OpenTracing
[using shims](/docs/migration/opentracing/).
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 5]
```diff
@@ -0,0 +1,431 @@
...
+
+The OpenTelemetry Operator is being implemented to auto-instrument Java and .NET
+code. The OTel Operator ([among other things](/docs/k8s-operator/#introduction))
+[supports injecting and configuring auto-instrumentation in .NET, Java, Python, and Node.js](/docs/k8s-operator/automatic/#configure-autoinstrumentation).
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:159]:
```suggestion
code. Among other things, the [OTel Operator](/docs/k8s-operator/)
supports injecting and [configuring auto-instrumentation](/docs/k8s-operator/automatic/#configure-autoinstrumentation) in .NET, Java, Python, and Node.js.
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 6]
```diff
@@ -0,0 +1,431 @@
...
+Iris hopes that Go auto-instrumentation will be available in the near-future.
+See [here](https://github.com/open-telemetry/opentelemetry-go-instrumentation)
+for more on the progress of auto-instrumentation in Go.
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:162]:
```suggestion
To track progress of auto-instrumentation in Go, see [OpenTelemetry Go Automatic Instrumentation](https://github.com/open-telemetry/opentelemetry-go-instrumentation).
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 7]
```diff
@@ -0,0 +1,431 @@
...
+### Can you share your experience around using the OTel Operator?
+
+The OTel Operator is only partially used in production, and is currently not
+available for everyone. Iris and her team really love the OTel Operator;
+however, it did take a bit of getting used to. Iris and her team found that
+there is a tight coupling between [cert-manager](https://cert-manager.io/) and
+the OTel Operator. They were not able to use our own custom certificates, and
+they did not support [cert-manager](https://cert-manager.io/) in their clusters,
+so they found it hard to use the Operator in our clusters.
+[They solved this by submitting a PR](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/760)!
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:184]:
```suggestion
They solved this by submitting a PR -- [opentelemetry-helm-charts PR #760](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/760)!
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 8]
```diff
@@ -0,0 +1,431 @@
...
+[log correlation](/docs/specs/otel/logs/#log-correlation), but it is something
+that Iris would like to explore further.
+
+Since logs are not yet stable, Iris doesn’t expect logging to go into production
+at Farfetch just yet. Farfetch has a huge volume of logs (more than traces), so
+they don’t want to start converting to OTel logging until things are more
+stable.
+
+> **_NOTE:
+> [Some parts of OTel logs are currently stable](/docs/specs/status/#logging)_**
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:206]:
```suggestion
> **Note**: some parts of OTel logs are stable. For details, see [Specification Status Summary](/docs/specs/status/#logging).
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 9]
```diff
@@ -0,0 +1,431 @@
...
+If you’d like to see my conversation with Iris in full, you can check out the
+video [here](https://youtu.be/9iaGG-YZw5I).
+
+If anyone would like to continue the conversation with Iris, please reach out to
+her in the
+[#otel-user-research](https://cloud-native.slack.com/archives/C01RT3MSWGZ) Slack
+channel!
+
+She will also be presenting at OTel in Practice on June 8th. Details can be
+found [here](https://shorturl.at/bqtxO).
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:403]:
```suggestion
She will also be presenting at [OTel in Practice on June 8th](https://shorturl.at/bqtxO).
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 10]
```diff
@@ -0,0 +1,431 @@
...
+[OTel Operator PR](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/760)
+approved and merged.
+
+## Additional Resources
+
+If you’d like to see my conversation with Iris in full, you can check out the
+video [here](https://youtu.be/9iaGG-YZw5I).
+
+If anyone would like to continue the conversation with Iris, please reach out to
+her in the
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:398]:
```suggestion
If anyone would like to continue the conversation with Iris, reach out to
her in the
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 11]
```diff
@@ -0,0 +1,431 @@
...
+
+Iris and her team were pleasantly surprised by the quick turnaround time on
+getting their
+[OTel Operator PR](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/760)
+approved and merged.
+
+## Additional Resources
+
+If you’d like to see my conversation with Iris in full, you can check out the
+video [here](https://youtu.be/9iaGG-YZw5I).
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:395]:
```suggestion
My conversation with Iris, in full, is [available on YouTube](https://youtu.be/9iaGG-YZw5I).
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 12]
```diff
@@ -0,0 +1,431 @@
...
+that they can maintain their stack on the clusters.
+
+### Have you dabbled in any of the OTel experimental features in Kubernetes?
+
+> **_NOTE: This question is referring to the ability for Kubernetes components
+> to emit OTLP traces which can then be consumed by the OTel Collector. More
+> info can be found
+> [here](https://kubernetes.io/docs/concepts/cluster-administration/system-traces/).
+> This feature is currently in beta, and was first introduced in Kubernetes 1.25
+> (release notes [here](https://sysdig.com/blog/kubernetes-1-25-whats-new/))._**
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:244]:
```suggestion
> This question is referring to the ability for Kubernetes components to emit
> OTLP traces which can then be consumed by the OTel Collector. For more info,
> see
> [Traces For Kubernetes System Components](https://kubernetes.io/docs/concepts/cluster-administration/system-traces/).
> This feature is currently in beta, and was first introduced in
> [Kubernetes 1.25](https://sysdig.com/blog/kubernetes-1-25-whats-new/).
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 13]
```diff
@@ -0,0 +1,431 @@
...
+or [redaction processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/redactionprocessor)),
+especially as they move to using OTel Logs, which will contain sensitive data that
+they won’t want to transmit to their Observability back-end. They currently, however,
+are only using the [batch processor](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md).
+
+### Are you aware of any teams using span events?
+
+> **_NOTE: A span event provides additional point-in-time information in a
+> trace. It’s basically a structured log within a span. For more on span events,
+> check out docs [here](/docs/concepts/signals/traces/#span-events)._**
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:308]:
```suggestion
> A [span event](/docs/concepts/signals/traces/#span-events) provides
> additional point-in-time information in a trace. It’s basically a structured
> log within a span.
```

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 14]
```diff
@@ -0,0 +1,431 @@
...
+  [CNCF Community Slack](https://communityinviter.com/apps/cloud-native/cncf)
+- Join our monthly
+  [End Users Discussion Group calls](/community/end-user/discussion-group/)
+- Join our [OTel in Practice sessions](/community/end-user/otel-in-practice/)
+- Sign up for one of our
+  [monthly interview/feedback sessions](/community/end-user/interviews-feedback/)
+- Join the
+  [OpenTelemetry group on LinkedIn](https://www.linkedin.com/groups/14081251)
+- Share your stories on the
+  [OpenTelemetry blog](https://github.com/open-telemetry/opentelemetry.io/blob/954103a7444d691db3967121f0f1cb194af1dccb/README.md#submitting-a-blog-post)
```
Comment (chalin) [inline content/en/blog/2023/end-user-q-and-a-03.md:426]:
```suggestion
- Share your stories on the
  [OpenTelemetry blog](https://github.com/open-telemetry/opentelemetry.io/blob/main/README.md#submitting-a-blog-post)
```

**Review by chalin — COMMENTED** [review]
Very nice! See inline for suggested edits.

File: content/en/blog/2023/end-user-q-and-a-03.md [hunk 15]
```diff
@@ -0,0 +1,432 @@
+---
+title: 'End-User Q&A Series: Using OTel at Farfetch'
+linkTitle: 'End-User Q&A: OTel at Farfetch'
+date: 2023-06-01
```
Comment (cartermp) [inline content/en/blog/2023/end-user-q-and-a-03.md:4]:
```suggestion
date: 2023-06-07
```

**Review by cartermp — APPROVED** [review]
Really great interview, this is fantastic stuff.

**Comment by chalin** [comment]
Hi @avillela - there are three remaining / pending of my comments. Just wondering if you were planning on addressing those too?

**Comment by avillela** [comment]
> Hi @avillela - there are three remaining / pending of my comments. Just wondering if you were planning on addressing those too?

@chalin my bad! They got buried in the middle there! I've taken care of them as well. 😄

**Review by chalin — APPROVED** [review]
✨

**Review by svrnm — APPROVED** [review]
awesome :-)

**Comment by avillela** [comment]
@chalin that latest `force-push` seems to have broken the build. I pulled in your changes and ran `rpm run tests` and it came up with a bunch of errors. Are you able to look into it?

**Comment by chalin** [comment]
@avillela - I've submitted a fix to the link, but until GH PRs are working again, we'll have to wait: https://www.githubstatus.com.

**Pull request merged**